CAP/gap/Derivations.gi

```diff
@@ -285,10 +285,7 @@ InstallMethod( AddDerivation,
     if preconditions_complete = false then
         for current_implementation in implementations_with_extra_filters do
 
-            current_list := CAP_INTERNAL_FIND_APPEARANCE_OF_SYMBOL_IN_FUNCTION( current_implementation[ 1 ], operations_in_graph, loop_multiplier );
-            current_list := List( current_list, i -> [ ValueGlobal( i[ 1 ] ), i[ 2 ] ] );
-            ## TODO: Maybe move the next line to AddDerivationToCAP
-            current_list := Concatenation( current_list, CAP_INTERNAL_FIND_APPEARANCE_OF_SYMBOL_IN_FUNCTION_FOR_MONOIDAL_CATEGORIES( current_implementation[ 1 ], loop_multiplier ) );
+            current_list := CAP_INTERNAL_FIND_APPEARANCE_OF_SYMBOL_IN_FUNCTION( current_implementation[ 1 ], operations_in_graph, loop_multiplier, CAP_INTERNAL_METHOD_RECORD_REPLACEMENTS );
```

Comment:
Since the function
`CAP_INTERNAL_FIND_APPEARANCE_OF_SYMBOL_IN_FUNCTION`
now takes an additional argument as its input, we need to adjust its documentation.

Reply:
Will do. I did not realize it has a documentation at all.